tests/fixtures/modules/dbt_config_parser_fixtures.py

```diff
@@ -42,26 +45,72 @@
         "model.main.stg_core_schema2__table2": {
             "schema": "analytics_engineering",
             "name": "stg_core_schema2__table2",
+            "unique_id": "model.main.stg_core_schema2__table2",
             "depends_on": {
                 "macros": [],
                 "nodes": [
                     "source.main.core_schema2.table2",
                     "source.main.core_schema2.table3",
+                    "seed.main.seed_buyer_country_overwrite",
```

Comment:
Maybe add the case of multiple (same) seed files and that duplicated inputs in the dagger config are avoided eventually.

Reply:
done !